### PR TITLE
Move semconv schema definition from build tools

### PR DESCRIPTION
**Pull request by lmolkova**
Fix https://github.com/open-telemetry/semantic-conventions/issues/916

(copy-paste from https://github.com/open-telemetry/build-tools/blob/15a0e831e0b0818ed666378adc85cba1fa66beef/semantic-conventions/semconv.schema.json)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#####################################################
```
Comment (lquerel) [inline .github/CODEOWNERS:1]: Do know if that can be a source of conflict but there is already a CODEOWNERS file at the root of this project.

**Review by lquerel — APPROVED** [review]
I left a comment regarding CODEOWNERS.

File: .github/CODEOWNERS (hunk 1 above)
Comment (lmolkova) [reply .github/CODEOWNERS:1]: oh, thanks! I didn't  realize there is one (and didn't know it could be in root). Fixing

**Comment by lquerel** [comment]
@lmolkova LGTM but I'd like to get @jsuereth feedback on this one to double the CODEOWNERS changes.

**Review by jsuereth — APPROVED** [review]

**Comment by jsuereth** [comment]
Reagarding: https://github.com/open-telemetry/community/blob/85f75d75789938ddfb42a9e3f251b85461cb210a/config.yaml.  We're moving to an git-ops like repository setup - They must have missed weaver in that, so we need to add it there.

**Pull request merged**